### PR TITLE
Fix ui issues

### PR DESCRIPTION
**Pull request by ShellyXueHan**
issue #32 
/bot-ignore-length

### CODE REVIEW

File: src/components/Auth/AuthModal.css [hunk 1]
```diff
@@ -49,3 +48,10 @@
 .modal-header .header-logo {
   height: 75px;
 }
+
+@media screen and (max-width: 500px) {
+  .auth-modal .modal-dialog {
+    width: 100%;
+    margin: 0;
+  }
+}
```
Comment (patricksimonian) [inline src/components/Auth/AuthModal.css]: new line

File: src/components/UI/Header.js [hunk 2]
```diff
@@ -21,20 +21,15 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
-import { Link } from 'react-router-dom';
 import AuthButton from '../Auth/AuthButton';
-import logo from './bcgovlogo.svg';
+import { APP_INFO } from '../../constants';
+import Title from './Title';
```
Comment (patricksimonian) [inline src/components/UI/Header.js]: I like the idea of abstracting this into a component but i think the name __Title__ is generic and could be misunderstood. Perhaps something like `Brand` or `Banner`?

File: src/components/UI/Title.css [hunk 3]
```diff
@@ -0,0 +1,15 @@
+.banner{
+  display: flex;
+  align-items: center;
+}
+
+.header-title {
+  font-family: 'Noto Sans','Calibri', 'Arial', 'Sans Serif';
```
Comment (patricksimonian) [inline src/components/UI/Title.css]: shouldn't be necessary to assign font family. This should be inherited higher up.

**Comment by ShellyXueHan** [comment]
@patricksimonian ready for review!

File: src/components/UI/Banner.js [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+import logo from './bcgovlogo.svg';
+import './Banner.css';
+
+export const Banner = ({ link = '', titleText }) => {
+  return (
+    <div className="banner">
+      <Link to={link}>
+        <img src={logo} className="header-logo" alt="logo" />
+      </Link>
+      <Textfit className="header-title" mode="multi">
```
Comment (patricksimonian) [inline src/components/UI/Banner.js:34]: I'd wrap this in an h1 maybe?

**Review by patricksimonian — APPROVED** [review]

File: src/components/UI/Banner.js (hunk 4 above)
Comment (ShellyXueHan) [reply src/components/UI/Banner.js:34]: using Textfit for responsiveness instead of h1. are you suggesting wrapping the text or the Textfit component?

**Pull request merged**